cluster.go

```diff
@@ -133,8 +179,9 @@ func elect(
 	}
 }
 
-func requestVotes(state * ServerState, voteChannels *[ClusterSize]chan Vote, winnerChannel chan bool) {
+func requestVotes(state * ServerState, voteChannels *[ClusterSize]chan Vote, onWinChannel chan bool) {
 	// send vote requests to other servers
+	//TODO: What if one of the servers is down -- then this will hang forever
```

Comment:
Oh shit! good point not sure how to handle that, we can talk about it with each other and Maria

Reply:
A cheap solution it to wrap everything in goroutines so that it doesn't block

Reply:
Yeah that seems fine